gsa/src/web/pages/policies/component.js

```diff
@@ -503,11 +524,12 @@ const PolicyComponent = ({
     handleInteraction();
   };
 
-  const handleImportPolicy = data => {
+  const handleImportPolicy = async data => {
     handleInteraction();
 
-    return gmp.policy
-      .import(data)
+    const readUploadedXml = readFileToText(data.xml_file);
+
+    return importPolicy(await readUploadedXml)
```

Comment:
Is there any reason for using async/await here? From looking at the code I would expect that is works without. This is not a criticism of your code or even an intention to rewrite it I am just curious :-) It's the first time we use async/await in our GSA code besides tests.